bottlecap/Cargo.toml

```diff
@@ -8,12 +8,13 @@ edition = "2021"
 [dependencies]
 figment = { version = "0.10.15", features = ["yaml", "env"] }
 log = "0.4.21"
-tokio = "1.37.0"
 serde = { version = "1.0.197", features = ["derive"], default-features = false }
+serde_json = "1.0.116"
```

Comment:
we should pin specific features or pass `--no-default-features` as per Brian's feedback [here](https://github.com/DataDog/datadog-lambda-extension/pull/212/files#diff-58ddbce581dc4225c4677dfa33a13bcef9019ac8157f1813250baa231725987aR18)

Reply:
Noted, thanks!